### ISSUE TITLE
Time Picker is not displaying properly within collapse

### ISSUE THREAD

**Issue by Sun-Sun** [Eonasdan/bootstrap-datetimepicker#1004]
Hello Please check this fiddle, The time picker is not displayed properly within the datepicker and the date picker itself is cut in half http://jsfiddle.net/0Ltv25o8/785/

**Comment by Eonasdan** [comment]
your fiddle doesn't work

**Comment by Sun-Sun** [comment]
I have updated the fiddle so it should work
http://jsfiddle.net/0Ltv25o8/792/

I will explain how to get to the broken calendar as well seen as my image icons are not available.

First you must click the append button on the top right.
A new row will appear, click the image closest to the input fields and a new row of fields will be added.
Here you will find the broken calendar


Sorry for the inconvenience as I am new to JSFiddle and forgot to update the fiddle before i sent it

**Comment by Eonasdan** [comment]
your fiddle still doesn't work very well. it seems like it's a CSS issue. perhaps you could get a test page setup?

**Comment by Sun-Sun** [comment]
Hello

Im not sure why it does not work for you because I just revisited the link and its working perfectly for me. In any sense however I have fixed this issue, Ill show you some screenshots of what happened for your reference so that you are aware of this problem at least. But I fixed it by displaying the calender side by side.
![bootstrapcalendar](https://cloud.githubusercontent.com/assets/10194820/7535041/80f3783a-f589-11e4-8f33-fec728fb857e.png)
![bootstrapcalendar2](https://cloud.githubusercontent.com/assets/10194820/7535042/80f3d94c-f589-11e4-8fd6-fcfa39aeb238.png)


I hope this wasn't too much of a bother
and Thank You for your help
